std-bits/table/src/main/java/org/enso/table/excel/ExcelRow.java

```diff
@@ -66,8 +71,10 @@ public boolean isEmpty(int column) {
   }
 
   public boolean isEmpty(int start, int end) {
-    for (int column = start; column <= end; column++) {
-      if (getCellType(get(column)) != CellType._NONE) {
+    int currentEnd = end == -1 ? getLastColumn() : end;
+    for (int column = start; column <= currentEnd; column++) {
+      CellType cellType = getCellType(get(column));
+      if (cellType != CellType._NONE && cellType != CellType.BLANK) {
         return false;
       }
```

Comment:
Won't it be simpler to 
```suggestion
    for (int column = start; column <= currentEnd; column++) {
       if (!isEmpty(column)) return false;
```

and then make the single argument overload:
```
public final boolean isEmpty(int column) {
      CellType cellType = getCellType(get(column));
      return cellType == CellType._NONE || cellType == CellType.BLANK;
}
```